### PR TITLE
Fixes #2086 - Creates an alumni contributor list

### LINKED ISSUE #2086: [content] Alumni page
I wonder if we need an alumni page for the site.
https://staging.webcompat.com/contributors
https://staging.webcompat.com/contributors/alumni
Some people have participated to this website meaningfully in the past.

See for example
* https://www.w3.org/People/
* https://www.w3.org/People/Alumni

### PR DESCRIPTION
**Pull request by karlcow**
This PR fixes issue #2086

## Proposed PR background

Trying to cater for people who contributed to the project in the past.

### CODE REVIEW

**Comment by karlcow** [comment]
This is a first pass at creating an alumni page. There are missing people and probably bios to double check with each individual.

**Comment by tagawa** [comment]
Looks like I'm listed twice in `alumni.html`. VIP treatment - achievement unlocked!!!

**Comment by karlcow** [comment]
@tagawa ahah. Special guest ;) once for Opera, once for Mozilla :p

**Comment by karlcow** [comment]
Managed to finally finish this. Or I hope. 😝 
Let's see.

File: webcompat/templates/contributors/alumni.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      </div>
+  </section>
+      </div>
+    </section>
+  </main>
+</div>
+{% endblock %}
+{%- block extrascripts -%}
+{%- if config.PRODUCTION or config.STAGING -%}
+<script src="{{ url_for('static', filename='js/dist/contributors.min.js')|bust_cache }}"></script>
```
Comment (miketaylr) [inline webcompat/templates/contributors/alumni.html:301]: I think @ksy36 will want to rebase after I merge this, to ensure the script references are correct.

SORRY KSENIA

File: webcompat/views.py [hunk 2]
```diff
@@ -455,6 +455,15 @@ def contributors():
     return render_template('contributors.html')
 
 
+@app.route('/contributors/alumni')
+@cache_policy(private=True, uri_max_age=0, must_revalidate=True)
+def alumni():
+    """Route to display contributors alumni page."""
+    if g.user:
```
Comment (miketaylr) [inline webcompat/views.py:462]: Not to self, this should just be a decorator. No need to change anything here, but silly to repeat ourselves so much in these routes (something I added, I'm sure).

Comment (karlcow) [reply webcompat/views.py:462]: yes! to the refactoring of g.user
Some context https://flask.palletsprojects.com/en/1.1.x/patterns/viewdecorators/

Comment (karlcow) [reply webcompat/views.py:462]: Filed #3329

**Pull request merged**